src/MacVim/MMWindowController.m

```diff
@@ -1329,6 +1333,26 @@ - (void)windowDidFailToExitFullScreen:(NSWindow *)window
 
 #endif // (MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_7)
 
+- (void)runAfterWindowPresentedUsingBlock:(void (^)(void))block
+{
+    if (windowPresented) { // no need to defer block, just run it now
+        block();
+        return;
+    }
+
+    // run block later
+    if (afterWindowPresentedQueue == nil) {
```

Comment:
There's little consistency in coding style (esp. `if`s). Probably want a style guide or a PR to do a mass-reformat a-la Postgresql/FreeBSD/etc..